nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -10,6 +10,11 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - Cinnamon has been updated to 5.6, see [the pull request](https://github.com/NixOS/nixpkgs/pull/201328#issue-1449910204) for what is changed.
 
+- The iputils package, which is installed by default, no longer provides the
```

Comment:
I think this is more of a thing for "backwards incompatibilities" than "highlights"?

Reply:
Yes indeed, you're right!
I moved that line to the "backwards incompatibilities" section.

Thanks for the review.